.github/workflows/ci.yml

```diff
@@ -103,6 +104,7 @@ jobs:
   test:
     name: test on ${{ matrix.python-version }}
     runs-on: ubuntu-latest
+    timeout-minutes: 10
```

Comment:
I accidentally created an infinite loop in a test by misusing one of the test Models, luckily was monitoring but don't want others to get caught